_posts/2019-10-28-MongoDB.md

```diff
@@ -72,21 +67,21 @@ passwordPrompt()
  
 This creates an admin user that has access to every aspect of any database in your deployment, so be cautious as to who you share that information with. Since MongoDB doesn’t enable authorization/authentication by default, even with the creation of an admin user, users will still have access to all aspects of every database in your deployment without the need to authenticate.
  
-To Enable authorization, first disconnect from the mongoshell and stop the mongo server (usually using ctrl^c in the terminal will do the trick). Then access the MongoD configuration file using your favourite editor. We will use $EDITOR for the purpose of this tutorial.
+To enable authorization, first disconnect from the mongoshell and stop the mongo server (usually using ctrl^c in the terminal will do the trick). Then access the MongoD configuration file using your favourite editor. We will use $EDITOR for the purpose of this tutorial.
```

Comment:
This comment with regards to `^C` is still missing.

Reply:
Updated both